src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -234,18 +231,13 @@ public ReadOnlyBorrowerRecords getBorrowerRecords() {
 
 
     @Override
-    public Optional<Borrower> getServingBorrower() {
-        // TODO
-        // return servingBorrower;
-        return Optional.of(new Borrower(new Name("Stub"), new Phone("12345"), new Email("mail@fakemail.co"),
-                new BorrowerId("K1234"))); // STUB
+    public Borrower getServingBorrower() {
+        return servingBorrower.get();
```

Comment:
Maybe we can add an assertion here that servingBorrower isPresent()

Reply:
Good point! Will add on!